perfscale_regression_ci/scripts/scalability/descheduler/common_func.sh

```diff
@@ -0,0 +1,50 @@
+
+function wait_for_pod_creation() {
+  COUNTER=0
+  creating=$(oc get pods -A | grep $1 | egrep -c -e "Pending|Creating|Error" )
+  while [ $creating -ne 0 ]; do
+    sleep 5
+    creating=$(oc get pods -A |  grep $1 | egrep -c -e "Pending|Creating|Error")
+    echo "$creating pods are still not running/completed"
+    COUNTER=$((COUNTER + 1))
+    if [ $COUNTER -ge 60 ]; then
+      echo "$creating pods are still not running/complete after 5 minutes"
+      break
+    fi
+  done
+}
+
+function uncordon_all_nodes() {
+  worker_nodes=$(oc get nodes -l node-role.kubernetes.io/worker= -o name)
+  for worker in ${worker_nodes}; do
+    oc adm uncordon $worker
+  done
+}
+
+
+function wait_for_descheduler_to_run() {
+  kube_desch_name=$(oc get KubeDescheduler -n openshift-kube-descheduler-operator -o name)
+  sched_seconds=$(oc get $kube_desch_name -n openshift-kube-descheduler-operator -o jsonpath='{.spec.deschedulingIntervalSeconds}')
+  echo "waiting for $sched_seconds seconds for descheduler to run"
+  sleep $sched_seconds
+}
+
+
+function get_descheduler_evicted() {
+  #desched_pod=$(oc get pods -n openshift-kube-descheduler-operator --no-headers -o name | grep cluster)
+  desched_pod=$(oc get pods -n openshift-kube-descheduler-operator --no-headers -o name | grep -v operator)
+  logs=$(oc logs $desched_pod -n openshift-kube-descheduler-operator --tail=10 |  grep 'Number of evicted pods')
+  echo "$logs"
+}
+
+function get_node_name() {
+  worker_name=$(echo $1 | rev | cut -d/ -f1 | rev)
+  echo "$worker_name"
+}
+
+function get_pod_count() {
```

Comment:
we might be able to combine this function with [count_running_pods ](https://github.com/openshift/svt/blob/6a0babd4e93c1107b4ac6c5de804a83bf0a5f481/perfscale_regression_ci/scripts/common.sh#L65) by adding a 3rd passed parameter with the pod name to grep on since we do have a project name 
Not sure if we really need the get_node_name, could use a line like [this one ](https://github.com/openshift/svt/blob/c3b7685341a2cba76b2827f1c689373335f81390/perfscale_regression_ci/scripts/scalability/pod-affinity-anti-affinity.sh#L33)instead to not need to use that function

Reply:
That would  affect the node and pod affinity tests (both of which use count_running_pods function).

Reply:
Think that is fine, we're going to need to make some updates to help reduce common code when possible